install-javafx.html

```diff
@@ -63,11 +63,11 @@ <h2>Run HelloWorld using JavaFX <span class="JFX_MAJOR">11</span></h2>
     </div>
 </div>
 
-<p>
-    <b>Note:</b> Additional modules are required for extended functionality.
+<div class="alert">
```

Comment:
Missing class. It should be `"alert alert-warning"` to get the yellow background.